src/rHelpProvider.ts

```diff
@@ -284,16 +291,26 @@ export class AliasProvider {
         this.allPackageAliases = null; 
 		const lim = '---vsc---'; // must match the lim used in R!
 		const re = new RegExp(`^.*?${lim}(.*)${lim}.*$`, 'ms');
-        const cmd = `${this.rPath} --silent --no-save --no-restore --slave -f "${this.rScriptFile}"`;
+        const tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'vscode-R-aliases-'));
+        const tempFile = path.join(tempDir, 'aliases.json');
+        const cmd = `${this.rPath} --silent --no-save --no-restore --slave -f "${this.rScriptFile}" > "${tempFile}"`;
```

Comment:
Does this work on Windows?

Reply:
It does work on my windows machine, and the `>` operator seems to be part of cmd/powershell, so I think it should work.

Reply:
Thanks for your confirmation!